src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -4292,6 +4330,10 @@ pre_envp(char *const envp[], char ***new_envp)
 	return rc;
 }
 
+/* Set this flag to avoid setup_fd_0_1_2() executed more than one time in case exec() fails due to
+ * wrong path.
+ */
+static bool setup_fd_0_1_2_done;
```

Comment:
are these changes unrelated to this PR? can they be moved to a different PR?

Reply:
This seems needed for daos build test in my impression. I can double-check.
execve() could fail when trying different paths to execute commands without absolute path.

Reply:
ok just wondering if they are related or not to the PR as it would be better to have them separate.

Reply:
You are right. The changes here are not necessary. I removed them. Thank you!